usecases/no2-monitoring/index.md

```diff
@@ -0,0 +1,68 @@
+# NO₂ monitoring
+
+TROPOMI is the name of a sensor on board of the Sentinel-5 Precursor (S5P) satellite, developed to monitor atmospheric chemistry.
+
+This use case analyses Sentinel 5P imagery, focusing in particular on NO₂ measurements. Compared to other variables measured by TROPOMI, NO₂ is of high interest not only because of its direct relation with environmental health, but also because the main sources are typically known, because it is not transported over long distances and because the total column values measured by TROPOMI are strong indication of the ground level values.
+
+NO₂ values will be analysed globally, over a full year (e.g., 2018). The analysis will allow for regridding to 10 km x 10 km grids, and allow the user to set threshold values for the quality flag andthe cloud cover. Noise and gaps due to quality flag or cloud cover will be removed by computing daily, 30-day smoothed values, using kernel smoothing of the time series. Interactions will involve (i) interaction with daily, color-coded maps including interactive and automatic time animation, (ii) the display of time series of smoothed, raw, and 30-day maximum values for locations selected by user mouse clicks, and (iii) the comparison of time series of user-selected hot spots with time series of averages for selected regions or for global averages.
```

Comment:
Other use case descriptions are written in present. Maybe switch the tense.

Reply:
Solved